content/blog/2021-06-28-numfocus.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "GNU Radio Joins NumFOCUS as an Affiliate Member"
+author: "Marc Lichtman"
+date: 2021-06-28T15:58:24-05:00
+sponsored: "0"
+aliases: ["blog/numfocus"]
+---
+
+GNU Radio is happy to join [NumFOCUS](https://numfocus.org/) as an Affiliated Project!
+<!--more-->
+[NumFOCUS](https://numfocus.org/) is a 501(c)(3) nonprofit that supports and promotes innovative open source scientific computing, with both fiscally sponsored and affiliated projects.  Some examples of NumFOCUS' [fiscally sponsored projects](https://numfocus.org/sponsored-projects) include NumPy, SciPy, Matplotlib, Pandas, IPython, Scikit-learn, and Julia.  Example [affiliated projects](https://numfocus.org/sponsored-projects/affiliated-projects) include Conda, Cython, Spyder, Numba, and now GNU Radio!
```

Comment:
is it *Affiliated Project* or *affiliated project*?

Reply:
They capitalize Affiliated Project in most spots on their website.  But in that paragraph it felt weird to capitalize it.